src/tscore/ink_mutex.cc

```diff
@@ -35,7 +35,7 @@ class x_pthread_mutexattr_t
   x_pthread_mutexattr_t()
   {
     pthread_mutexattr_init(&attr);
-#ifndef POSIX_THREAD_10031c
+#if HAVE_PTHREAD_MUTEXATTR_SETPSHARED
     pthread_mutexattr_setpshared(&attr, PTHREAD_PROCESS_SHARED);
```

Comment:
I am not sure if this is the correct change here. The old code essentially disabled this call and it dates to the original SVN import. 

Should it continue to be ignored, and if so, should it be deleted?

Reply:
Continued investigation of the old SVN import indicates that this dates to at least v2.0.1(!) and SVN r831143 (the initial open-source import) and that `POSIX_THREAD_10031c` was defined on a Linux, FreeBSD or Darwin system.

Reply:
I do not think we should be setting mutexes pshared at all.  I don't think it matters anyways unless the mutexes are in shared memory regions.  I think we could just delete this code.